examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,63 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+
+const gasLimit = 500000;
+if (stdlib.connector == 'ETH') {minter.setGasLimit(gasLimit)};
+if (stdlib.connector == 'ETH') {receiver.setGasLimit(gasLimit)};
+
+const mintAddr = minter.getAddress();
+console.log(`Your address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who) => fmt(await stdlib.balanceOf(who));
+const bal = await getBal(minter);
+console.log(`Minter starting balance: ${bal}.`);
+
+const mintNFT = async () => {
```

Comment:
I think `mintNFT` and `transferNFT` should be abstracted more. I'd make any value that someone would want to customize a parameter. For example, I would make the minter account a parameter of this function as well as an object of token properties. 

```javascript
const mintNFT = async (minter, opts) => { ... }
// where opts : { name, symbol, supply, au, c, ... }

await mintNFT(minter, {});
```
I think default assignments would work well here, so if the caller of `mintNFT` doesn't care about a particular field, like `reserve`, the function will use `null` by default.